plotly/tests/test_core/test_offline/test_offline.py

```diff
@@ -0,0 +1,18 @@
+"""
+test__offline
+
+"""
+from __future__ import absolute_import
+
+import os
+
+import plotly
+
+
+def test_downloading_file_saves_it_to_the_disk():
```

Comment:
:cow2: , might be a good habit to start using `TestCase` instead of just bare tests.